spec/unit/plugin_manager/offline_plugin_packager_spec.rb

```diff
@@ -40,6 +41,8 @@ def retrieve_dependencies_gems(path)
   let(:temporary_dir) { Stud::Temporary.pathname }
   let(:target) { ::File.join(temporary_dir, "my-pack.zip")}
   let(:extract_to) { Stud::Temporary.pathname }
+  let(:retries_count) { 20 }
```

Comment:
What do you think of bumping this to 30 or 50? the first few retries are subsecond, so 20 means around 10-15 seconds of retry time, which may not be enough?